app/views/claims/support/schools/check.html.erb

```diff
@@ -5,7 +5,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= form_with(model: @school, url: claims_support_schools_path, method: "post", data: { turbo: false }) do |f| %>
-      <%= f.hidden_field :urn, value: @school.urn %>
+      <%= f.hidden_field :search_urn, value: @school.urn %>
```

Comment:
Any reason why you renamed this field/param from `urn` to `search_urn`?

I have no strong opinion either way – just thought `urn` was probably a more accurate name. By the time they get to the 'check' stage, they're not 'searching' for a school any more.

I also see you've stuck with `code` on the provider form, rather than renaming that field to `search_code`.

Reply:
This is because in the `new` form page, the `:urn` is used to for the search (showing the name of the school) and the JS adding a `search_urn` field.
Originally the JS added a selection field which led to the following code in the controller:
```
def urn_param
  params.dig(:selection, :urn) || params.dig(:school, :urn)
end
```
Changing this to `search_urn` means we only need to use the following code in the controller:
```
def urn_param
  params.dig(:school, :search_urn)
end
```
Minimising code and confusion (hopefully).